modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -31,14 +32,60 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
+.. Add the `DestinationRule` CR to a file called _destination-rule-threescale-saas-backend.yml:_
 +
-.Custom resource for 3scale Hosted system
+.DestinationRule CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-backend
+spec:
+  host: su1.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: su1.3scale.net
+----
+
+. Apply and save the external `ServiceEntry` CR for the 3scale Hosted backend to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-backend.yml>
+----
+
+.. Alternatively, apply the external `ServiceEntry` CR for the 3scale Hosted backend to your cluster without first saving the file, by running the following command:
++
+[source,terminal]
+----
+$ echo -n "<service-entry-threescale-saas-backend.yml>" | oc apply -f -
+----
+
+. Apply and save the external `DestinationRule` CR for the 3scale Hosted backend to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <destination-rule-threescale-saas-backend.yml>
```

Comment:
The `service-entry-threescale-saas-backend.yml` reference in this command is passed literally to the command line and it is not a variable. This is because the preceding substep requested that the file is saved explicitly using this file name. You could remove the parenthesis:

> $ oc apply -f destination-rule-threescale-saas-backend.yml